apps/aecore/include/txs.hrl

```diff
@@ -0,0 +1,77 @@
+-type(pubkey() :: binary()).
+
+-record(coinbase_tx, {account = <<>> :: pubkey(),
+                      nonce = 0      :: non_neg_integer()}).
+
+-record(spend_tx, {from = <<>> :: pubkey(),
+                   to = <<>>   :: pubkey(),
+                   amount = 0  :: non_neg_integer(),
+                   fee = 0     :: non_neg_integer(),
+                   nonce = 0   :: non_neg_integer()}).
+
+
+%% Channel-related transactions
+
+-record(channel_new_tx, {id = 0          :: non_neg_integer(),
+                         account1 = <<>> :: pubkey(),
+                         account2 = <<>> :: pubkey(),
+                         amount1 = 0     :: non_neg_integer(),
+                         amount2 = 0     :: non_neg_integer(),
+                         fee = 0         :: non_neg_integer(),
+                         nonce = 0       :: non_neg_integer(),
+                         delay = 0       :: non_neg_integer()}).
+
+-record(channel_grow_tx, {id = 0          :: non_neg_integer(),
+                          account1 = <<>> :: pubkey(),
+                          account2 = <<>> :: pubkey(),
+                          amount1 = 0     :: non_neg_integer(),
+                          amount2 = 0     :: non_neg_integer(),
+                          fee = 0         :: non_neg_integer(),
+                          nonce = 0       :: non_neg_integer(),
+                          delay = 0       :: non_neg_integer()}).
+
+-record(channel_team_close_tx, {id = 0          :: non_neg_integer(),
+                                account1 = <<>> :: pubkey(),
+                                account2 = <<>> :: pubkey(),
+                                amount = 0      :: pubkey(),
+                                fee = 0         :: non_neg_integer(),
+                                nonce = 0       :: non_neg_integer()}).
+
+-record(channel_solo_close_tx, {id = 0         :: non_neg_integer(),
+                                account = <<>> :: pubkey(),
+                                fee = 0        :: non_neg_integer(),
+                                nonce = 0      :: non_neg_integer()}).
+
+-record(channel_slash_tx, {id = 0         :: non_neg_integer(),
```

Comment:
can we improve naming? slash is not saying anything? is anyone using it and this is why we use slash? Maybe: Overload?

Reply:
The indent I prefer is how my Emacs is indenting if you leave the opening brace hanging.

Reply:
Slash is a [cryptocurrency term](https://medium.com/@VitalikButerin/minimal-slashing-conditions-20f0b500fc6c) so I would leave it. We'll evaluate the transaction itself when we get to channels.